### ISSUE TITLE
Drug Cohorts

### ISSUE THREAD

**Issue by Braventos** [oxford-pharmacoepi/CohortSymmetry#9]
At the moment getPSSA() is the only function that users need to interact with. getPSSA() allows users to either reference an existing table with drugs (specified with the 'cohort_table' parameter) or create a new one by specifying the drugs of interest using generateDrugCohortPSSA() (with the table named after the 'table_name' parameter). 

For the package, I think getPSSA() should only be used to point to existing tables. If we want to keep the functionality of creating drug cohorts, I'd use generateDrugCohortPSSA() as a preliminary step before using getPSSA(). At the moment both drugs are instantiated in the same table. However, if we want to use the same cohorts for other analyses, it might be better to use separate tables for each drug. 

To sum up:
- Should 'getPSSA()' exclusively reference existing tables?
- Do we want to maintain the functionality of creating drug tables, or is this better left to the user?
- Should we have tables containing both drugs, or opt for separate tables for each drug?

What do you think? @xihang-chen @edward-burn  @daniellenewby

**Comment by edward-burn** [comment]
It would be nice to align with cohort survival I think, with PSSA having it's primary function something like estimatePSSA  https://darwin-eu-dev.github.io/CohortSurvival/reference/estimateSingleEventSurvival.html

Users can then instantiate drug cohorts using the DrugUtilisation package directly

**Comment by Braventos** [comment]
Thanks! Looking at other packages, we could rename getPSSA() to estimatePSSA(), and replace the parameters index, marker, cohort_table, and table name for IndexCohortTable, IndexCohortId, MarkerCohortTable, MarkerCohortId. 

At the moment start and end dates are defined as part of getPSSA(). I believe dates are handled outside the main function in CohortSurvival. We could consider moving this functionality to generateDrugCohortPSSA().
